src/ratings-and-reviews/index.jsx

```diff
@@ -19,18 +19,37 @@ const reviewsTotal = (reviews) => {
   return total;
 };
 
+/** @param {Object}  */
```

Comment:
It would need to be `/** @param {Object} reviews */` - you name it for the parameter